apps/site/src/components/pages/wordpress/why-develop.tsx

```diff
@@ -0,0 +1,139 @@
+import { Box, Grid, Typography } from "@mui/material";
+
+import { AtomSimpleIcon, CityIcon, WandMagicSparklesIcon } from "../../icons";
+import { IconSection } from "./icon-section";
+
+export const WhyDevelop = () => {
+  return (
+    <Box
+      sx={{
+        fontFamily: "Inter",
+        position: "relative",
+        zIndex: 1,
+        pb: 15,
+      }}
+    >
+      <Box
+        sx={{
+          position: "absolute",
+          width: "60%",
+          maxWidth: 660,
+          height: 75,
+          top: "-11rem",
+          left: "50%",
+          transform: "translateX(-50%)",
+          background: "#0500FF",
+          opacity: 0.5,
+          filter: "blur(100px)",
+        }}
+      />
+
+      <Box
+        sx={{
+          margin: "auto",
+          width: "calc(100% - 2rem)",
+          position: "absolute",
+          bottom: "100%",
+          left: "1rem",
+          zIndex: 1,
+        }}
+      >
+        <Box
```

Comment:
![29F698DA-262D-45CB-9A74-97CF294D5317](https://user-images.githubusercontent.com/6226576/207741373-933b711f-1294-40f8-b873-d35ecae79a5a.PNG)

On mobile-width viewports, the "Developing <strong>Þ</strong> blocks has a number of benefits..." text bleeds over the transparent box overlay of the gradient background beneath. We should avoid this.

Reply:
This shouldn't be happening after bf9946becb1a7ef4e82ecb54efa419f06a84bc21, can you check again please?